packages/vscode-extension/messages.json

```diff
@@ -18,8 +18,8 @@
 	"COMMAND_TRIGGERED": "'{0}' has been triggered.",
 	"COMMAND_WARNING": "'{0}' has finished.",
 
-	"COMPARE_FILE_COMPARE_FILE_WITH_ACCOUNT_FILE": "Compare with Account File",
 	"COMPARE_FILE_COMPARING_FILE": "Comparing file...",
+	"COMPARE_FILE_EDITOR_LABEL": "{0} (Account ⟷  Local)",
```

Comment:
Seems to be an extra space between  ⟷  Local

Reply:
Intentional because without that extra space, it looks like the right there is no space between the right arrow and Local Copy.

Reply:
oh great. then all good!:)